package.json

```diff
@@ -117,6 +118,143 @@
                 "language": "xquery-ml",
                 "path": "./snippets/snippets.json"
             }
+        ],
+        "breakpoints": [
+            {
+                "language": "javascript"
+            }
+        ],
+        "debuggers": [
+            {
+                "type": "ml-jsdebugger",
+                "label": "MarkLogic JS Debugger",
+                "program": "./dist/jsDebug.js",
+                "runtime": "node",
+                "configurationAttributes": {
+                    "launch": {
+                        "required": [
+                            "hostname"
+                        ],
+                        "properties": {
+                            "hostname": {
+                                "type": "string",
+                                "description": "hostname of MarkLogic server",
+                                "default": ""
+                            },
+                            "username": {
+                                "type": "string",
+                                "description": "Username.",
+                                "default": ""
+                            },
+                            "password": {
```

Comment:
Where is the password stored? In plain text in the configuration file won’t work.

Reply:
User can choose to put password in config file. If they don't,  extension will ask for the password for each run. (maybe we should remove the option for user to put down password?)